static/src/javascripts/bootstraps/standard/emailEmbeds.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+
+const initEmbedResize = () => {
+    const allIframes: HTMLIFrameElement[] = ([
+        ...document.querySelectorAll('.element-embed > .email-sub__iframe'),
+    ]: any);
+
+    window.addEventListener('message', event => {
+        const iframes: HTMLIFrameElement[] = allIframes.filter(i => {
+            try {
+                return i.src === event.source.location.href;
```

Comment:
I wonder, is this still working if the iframe redirects to a new URL, like once the form data has been posted?
I don't think it's a deal breaker as it makes sense for the iframe to keep its size after the form is submitted, but it's more to satisfy my curiosity.